frontend/src/app/services/canvas.service.ts

```diff
@@ -56,6 +56,19 @@ export class CanvasService {
     return savedPost;
   }
 
+  async createListPost(post: Post) {
+    const savedPost = await this.postService.create(post);
+    this.socketService.emit(SocketEvent.POST_CREATE, savedPost);
```

Comment:
If we emit this event, won't it show the post on other users' canvas? But, it should stay only inside the list modal.